### ISSUE TITLE
ko.mapping.updateFromJS and observable returned from create callback

### ISSUE THREAD

**Issue by Anber** [SteveSanderson/knockout.mapping#26]
Sample code:

```
var viewModel = function(options) {
    ko.mapping.fromJS(options.data, {}, this);

    this.func = function() { return true; };
};

var dataForInit = {
    "id": 1,
    "entry": {
        "value": 1
    }
};

var dataForUpdate = {
    "id": 1,
    "entry": {
        "value": 2
    }
};

var model = ko.mapping.fromJS(dataForInit, {
    entry: {
        create: function(options) {
            return ko.observable(new viewModel(options));
        }
    }
});

ko.mapping.updateFromJS(model, dataForUpdate);

value_of(model.entry().value()).should_be(2);
```

possible fix https://github.com/Anber/knockout.mapping/commit/c173fe62063556a4653806425f146354411ba5f0

**Comment by sagacity** [comment]
Hi Anber, this is already fixed in the 2.0 beta. Can you confirm it works for you as well?

Available here:
https://github.com/SteveSanderson/knockout.mapping/blob/2.0/knockout.mapping.js

**Comment by Anber** [comment]
Yes, it works.

Thanks.
